selfdrive/controls/lib/events.py

```diff
@@ -331,7 +331,7 @@ def wrong_car_mode_alert(CP, sm, metric):
       "TAKE CONTROL",
       "Lane Departure Detected",
       AlertStatus.userPrompt, AlertSize.mid,
-      Priority.LOW, VisualAlert.steerRequired, AudibleAlert.chimePrompt, 1., 2., 3.),
+      Priority.LOW, VisualAlert.steerRequired, AudibleAlert.chimePrompt1, 1., 2., 3.),
```

Comment:
why did this change?

Reply:
I had that just to trigger an exception, that will be removed